app/serializers/new_mapping_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class NewMappingSerializer < ActiveModel::Serializer
+  embed :ids, include: true
+  attributes :id,
+             :xloc,
+             :yloc,
+             :created_at,
+             :updated_at
+  has_one :user, serializer: NewUserSerializer
+  has_one :map, serializer: NewMapSerializer
+  has_one :mappable, polymorphic: true ##?
```

Comment:
gotta figure out how to handle this polymoprhic so that it uses the right serializers in both cases...